jobs-api-server/src/test/java/org/zowe/jobs/services/zosmf/GetJobsZosmfRequestRunnerTest.java

```diff
@@ -52,6 +52,23 @@ public void get_output_jobs_should_call_zosmf_and_parse_response_correctly() thr
 
         test_getJobs(JobStatus.OUTPUT, Arrays.asList(job1, job4));
     }
+    
+    @Test
+    public void get_output_jobs_null_owner_should_call_zosmf_and_parse_response_correctly() throws Exception {
+        Job job1 = createJob("STC16867", "ZOEJC", "IZUSVR", "STC", JobStatus.OUTPUT, "Job is on the hard copy queue",
```

Comment:
Minor point, but I wonder if it is worth createing a new zosmf_getJobs_singleuser_Response.json type thing and change the sample data to have the same owner for all the jobs. Otherwise it's a little confusing from the unit test and looks like null owner defaults to *, not the logged in user?